core/audits/script-treemap-data.js

```diff
@@ -167,8 +167,10 @@ class ScriptTreemapDataAudit extends Audit {
     const nodes = [];
     /** @type {Map<string, LH.Treemap.Node>} */
     const htmlNodesByFrameId = new Map();
-    const bundles = await JSBundles.request(artifacts, context);
-    const duplicationByPath = await ModuleDuplication.request(artifacts, context);
+    const [bundles, duplicationByPath] = await Promise.all([
+      JSBundles.request(artifacts, context),
+      ModuleDuplication.request(artifacts, context),
+    ]);
 
```

Comment:
> these (and the ones I did not comment on) don't do any i/o work (look at JsBundles.compute_ and ModuleDuplication.compute_) - and since JS is _concurrent_ but not _parallel_ (w/o using workers....) there is no benefit to running like this

I am extremely sorry @connorjclark. Yeah, it seems useless. Shall I revert that change in this part?

Reply:
🙌🏽